src/views/catalogi/CatalogiDetails.vue

```diff
@@ -178,65 +178,57 @@ export default {
 			handler(newCatalogiItem, oldCatalogiItem) {
 				if (!this.upToDate || JSON.stringify(newCatalogiItem) !== JSON.stringify(oldCatalogiItem)) {
 					this.catalogi = newCatalogiItem
-					newCatalogiItem && this.fetchData(newCatalogiItem?.id)
+					if (newCatalogiItem) {
+						this.loading = true
+						catalogiStore.getCatalog(newCatalogiItem.id)
```

Comment:
`getCatalog` bestaat niet, gebruik `getOneCatalogi`
```suggestion
						catalogiStore.getOneCatalogi(newCatalogiItem.id)
```

Reply:
@SudoThijn zou jij deze kunnen oppakken?